release-engineering/roadmaps/anago_krel_migration.md

```diff
@@ -0,0 +1,111 @@
+# Krel - Anago Migration Roadmap
+
+_Moving away from running bash in production in k/release_
+
+## Objectives
+
+This roadmap defines a strategy for achieving two primary goals: migrating
+exchangeable bits of bash code within Anago to krel creating a Golang native
+replacement in krel that relies only on krel (stage/release) subcommands and
+enables us to remove anago completely.
```

Comment:
Looks like some sentences here got merged together

Reply:
Yep, fixed that sentence. :+1: